favorites/forms.py

```diff
@@ -0,0 +1,5 @@
+from django import forms
+
+class NewFavoriteForm(forms.Form):
```

Comment:
You're not using this form anywhere. Try it yourself; delete this file and the import in `views.py` and everything still works. Make sure that makes sense to you.

Reply:
I think I understand the redundancy. So I could have a view that displays a form i.e. `class NewFavoriteView(FormView)` which would use `NewFavoriteForm` or I could just use `CreateView` which is a view that automatically displays a form for creating the specified object with specified fields like so: `class NewFavoriteView(CreateView)` and then I wouldn't need to create the separate form. So because I'm using CreateView, it's not using my form at all. Right?

Reply:
Pretty much! `CreateView` lets you set the form directly by specifying `form_class` as an attribute, but if you don't do that it uses [`modelform_factory`](https://docs.djangoproject.com/en/1.11/ref/forms/models/#django.forms.models.modelform_factory) to inspect the model and the fields you have specified, and magically build a form for you. [This is the actual Django code](https://github.com/django/django/blob/1.11.6/django/views/generic/edit.py#L103).